### ISSUE TITLE
Submit letter of interest for Coding It Forward

### ISSUE THREAD

**Issue by amenity** [cityofaustin/atd-data-tech#4639]
Docs [here](https://drive.google.com/drive/u/0/folders/1fVXL1i8okeoTQ6K9fRGBWbOU86ySqJUx).

**Comment by amenity** [comment]
@mateoclarke @johnclary - sticking this in this sprint since they want it next week. Should I set up a call to discuss what projects to propose?

**Comment by johnclary** [comment]
thanks @amenity. Yes, please. Would you and @mateoclarke dig up what we submitted to the Github charity hours? Thinking that we could craft easily craft projects for a designer, developer, and data scientist, assuming we have bandwidth to manage their work? Let's include @SurbhiBakshi as well.

**Comment by amenity** [comment]
@SurbhiBakshi - documents are in the Drive folder above will help explain the organization and proposal. 

Tl;dr we have a wonderful opportunity to bring on some very-vetted, young, diverse interns in Summer 2021! The key will be choosing projects that will be fulfilling to them while supporting our organizational priorities... and ensuring that the interns will be able to operate as true collaborators without excessive supervision on the part of our team.
